ui/src/dashboards/components/TableOptions.tsx

```diff
@@ -107,6 +109,12 @@ export class TableOptions extends PureComponent<Props, {}> {
 
   handleToggleTextWrapping = () => {}
 
+  handleToggleFixFirstColumn = () => {
+    const {handleUpdateTableOptions, tableOptions} = this.props
+    const fixFirstColumn = !tableOptions.fixFirstColumn
```

Comment:
I wonder if you can get this event from the element itself rather than setting value to be the reverse of the previous value. Maybe it doesn't matter, I'm not sure. But if you end up using something like a checkbox as the element you toggle at the end, I would expect the function that handles it to use the value coming in with the event rather than toggling the value to be the opposite of what it used to be.